grunt/connect.js

```diff
@@ -3,8 +3,8 @@ module.exports = {
     options: {
       hostname: '*',
       port: process.env.PORT || 6789,
-      base: ['.', 'src'],
-      directory: 'src',
+      base: ['.', 'dist'],
```

Comment:
with browserify now set up, there's no need to do this since `index.html` is no longer importing libraries from our `node_modules` directory. move back to:

```
options: {
    hostname: '*',
    port: process.env.PORT || 6789,
    base: 'dist/',
    livereload: true
}
```